app/products/calls/screens/call_screen/call_screen.tsx

```diff
@@ -499,11 +512,22 @@ const CallScreen = ({
                         onPress={switchToThread}
                         text={callThreadOptionTitle}
                     />
+                    {
+                        ccAvailable &&
+                        <SlideUpPanelItem
+                            leftIcon='closed-caption-outline'
+                            onPress={toggleCC}
+                            text={showCC ? hideCCTitle : showCCTitle}
+                        />
+                    }
                 </View>
             );
         };
 
-        const items = isHost && EnableRecordings ? 3 : 2;
+        let items = isHost && EnableRecordings ? 3 : 2;
+        if (ccAvailable) {
+            items++;
+        }
```

Comment:
Curious, would it work if we summed the booleans? :p

Reply:
You're kidding, right? :)

Reply:
Sure...